packages/react/modules/components/toggle-feature/toggle-feature.ts

```diff
@@ -32,8 +31,13 @@ const ToggleFeature = (props: Props): React.ReactNode => {
     if (props.toggledComponent)
       return React.createElement(props.toggledComponent);
 
-    if (props.children && !isEmptyChildren(props.children))
-      return React.Children.only(props.children);
+    if (props.children) {
+      if (typeof props.children === 'function')
+        return props.children({
+          isFeatureEnabled: props.isFeatureEnabled,
+        });
+      return React.Children.only<React.ReactNode>(props.children);
```

Comment:
I'm not sure what the `isEmptyChildren` check is for, so I left it out as we want to render the children here.

Please correct me if I got this wrong.

Reply:
Can't fully recall. Was something people recommended doing. I am fine with leaving it out!

Reply:
True.